src/taco-dependency-installer/installers/androidPackagesInstaller.ts

```diff
@@ -77,26 +82,25 @@ class AndroidPackagesInstaller extends InstallerBase {
         }
     }
 
-    private getAvailableAndroidPackages(): Q.Promise<any> {
-        var deferred: Q.Deferred<any> = Q.defer();
+    private executeAndroidCommand(name: string, args: string[]): IExecuteCommandResult {
```

Comment:
I changed the signature to:
executeAndroidCommand(args: string[], nameForTelemetry: string,
        shouldAutomaticallyAcceptLicenseTerms: boolean): Q.Promise<string>
And move the accepting license terms logic inside given that we only have two callers, I think it's clearer to have all parameters mandatory.

What are you referring to with: expectPrompt: boolean?